packer/packer.json

```diff
@@ -47,6 +47,37 @@
         "kubernetes_version": "{{user `kubernetes_version`}}",
         "kubernetes_cni_version": "{{user `kubernetes_cni_version`}}"
       }
+    },
+    {
```

Comment:
I would like for us to maintain parity with other clouds with regard to baseline support (ie. Ubuntu and CentOS).  Happy to take CentOS in a follow-up PR, but we should Issue it.

Reply:
I'm looking through CentOS images and I don't see any "official" CentOS images:

```
$ az vm image list -f CentOS --all | jq -r '.[].publisher' | sort | uniq
MicrosoftRServer
OpenLogic
RightScaleLinux
batch
center-for-internet-security-inc
cloudera
cloudhub-technologies
cognosys
ingrammicro
saltstack
scalegrid
tokyosystemhouse
tunnelbiz
ubercloud
wmspanel
```

any idea what we should do there?

Reply:
Looks like the Azure supported version is from OpenLogic.